package.json

```diff
@@ -12,6 +12,7 @@
   "workspaces": [
     "common",
     "cli",
+    "did-sdk",
```

Comment:
added this to the monorepo so you can install all dependencies with `yarn` from root. added a bunch to the root yarn.lock